components/colorarea/index.css

```diff
@@ -9,87 +9,89 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 .spectrum-ColorArea {
-  --spectrum-colorarea-border-radius: var(--spectrum-color-area-border-rounding);
-  --spectrum-colorarea-border-color: rgba(0, 0, 0, 10%); /* TODO replace with token --spectrum-color-area-border-color and --spectrum-color-area-border-opacity using RGBA function */
-  --spectrum-colorarea-disabled-background-color: var(--spectrum-disabled-background-color);
-  --spectrum-colorarea-border-width: var(--spectrum-color-area-border-width);
-  --spectrum-colorarea-height: var(--spectrum-color-area-height);
-  --spectrum-colorarea-width: var(--spectrum-color-area-width);
-  --spectrum-colorarea-min-width: var(--spectrum-color-area-minimum-width);
-  --spectrum-colorarea-min-height: var(--spectrum-color-area-minimum-height);
+	--spectrum-colorarea-border-radius: var(--spectrum-color-area-border-rounding);
+	--spectrum-colorarea-border-color-rgb: var(--spectrum-gray-1000-rgb);
```

Comment:
Should we keep the TODO comment? Technically, the designs are asking for `gray-1000`, not `gray-1000-rgb`. Once `gray-1000` gets updated without that rgba() color function, we could remove the use of that custom property `--spectrum-colorarea-border-color-rgb` to "properly" use the `gray-1000`.

Reply:
That makes sense to me — I dropped in the same component that I used in the `colorwheel` since it's the same pattern. ✨

Reply:
> Something I just thought of- I wonder if we should move the forced-colors query. I would have to check with Rise and Josh (or dig through old PRs), but I think when we were doing these components before, we were trying to move the forced-colors block to the bottom of the CSS file to work with the cascade.

I went ahead and moved this down — it's a fairly consistent pattern and was the case when I did WHC work recently. Happy to revert if @jawinn or @rise-erpelding would like though. 🙂